addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -5079,12 +5079,7 @@ X[]
                                                 '<td>ef</td>' +
                                             '</tr></tbody></table>',
                             stepFunction: async editor => editor.execCommand('applyColor', 'aquamarine', 'color'),
-                            contentAfterEdit: '<p>a<font style="color: aquamarine;">[bc</font></p>' +
-                                            '<table class="o_selected_table"><tbody><tr>' +
-                                                '<td class="o_selected_td" style="color: aquamarine;">a]b</td>' +
-                                                '<td class="o_selected_td" style="color: aquamarine;">cd</td>' +
-                                                '<td class="o_selected_td" style="color: aquamarine;">ef</td>' +
-                                            '</tr></tbody></table>',
+                            contentAfterEdit: '<p>a<font style="color: aquamarine;">[bc</font></p><table class="o_selected_table"><tbody><tr><td class="o_selected_td"><font style="color: aquamarine;">a]b</font></td><td class="o_selected_td"><font style="color: aquamarine;">cd</font></td><td class="o_selected_td"><font style="color: aquamarine;">ef</font></td></tr></tbody></table>',
```

Comment:
This is hard to read. I suggest using the `unformat` util. This applies to all the other tests.

Reply:
Thanks, didn't know about this (I would have suffered a lot by hand)